kerchunk/zarr.py

```diff
@@ -1,11 +1,16 @@
 import fsspec
 from fsspec.implementations.reference import LazyReferenceMapper
 
-from kerchunk.utils import class_factory
+from kerchunk.utils import consolidate, class_factory
 
 
 def single_zarr(
-    uri_or_store, storage_options=None, inline_threshold=100, inline=None, out=None
+    uri_or_store,
+    storage_options=None,
+    inline_threshold=100,
+    inline=None,
+    out=None,
+    translate=False,
```

Comment:
I think it should just always be True, no need for the kwarg. (Also "translate" is the method on most scanning classes, so would be confusing).
A good argument to optionally include would be an output; at the moment we have out is None (return JSON), or a parquet set; if it were a .json path, we should write to it.

Reply:
Apologies pushed an old commit. renamed it to consolidate

Reply:
I'll take a look at the write to json